CONTRIBUTING.md

```diff
@@ -115,4 +115,22 @@ $ node_modules/.bin/gulp serve --server=<che_api_url>
 
 
 ### Che server a.k.a WS master
-To be provided soon.
+There is a [devfile](https://github.com/eclipse/che/blob/master/devfile.yaml) for development of Che server in Che.
+To build Che one may run a predefined build task from the devfile.
+
+Starting Che master requires some manual steps.
+Open a terminal in runtime container (`che-server-runtime`) and perform:
+ - First, set `CHE_HOME` environment variable with absolute path to parent folder of Che master's Tomcat.
+   It might look like `/projects/che/assembly/assembly-main/target/eclipse-che-*-SNAPSHOT/eclipse-che-*-SNAPSHOT`.
+ - Then set `CHE_HOST` with the endpoint of new Che master. For the devfile it should be `che-dev`
```

Comment:
Sorry, but I do not really understand the following formulating:
```
For the devfile it should be `che-dev`
For example, for Minishift it might be `http://che-che.192.168.99.102.nip.io`.
```
Could you explain please? If I use minishift and devfile, should it be `che-dev` or ``http://che-che.192.168.99.102.nip.io``?

Reply:
Yes, you are right it is a mistake. Will fix.